slot_attention/slot_attention.py

```diff
@@ -8,9 +8,10 @@ def __init__(self, num_slots, dim, iters = 3, eps = 1e-8, hidden_dim = 128):
         self.iters = iters
         self.eps = eps
         self.scale = dim ** -0.5
-
-        self.slots_mu = nn.Parameter(torch.randn(1, 1, dim))
-        self.slots_sigma = nn.Parameter(torch.randn(1, 1, dim) * self.scale)
+        
+        _glorot = (6/(1+dim))**0.5
+        self.slots_mu = nn.Parameter(torch.rand(1, 1, dim)*2*_glorot - _glorot)
```

Comment:
could you make use of https://pytorch.org/docs/stable/nn.init.html#torch.nn.init.xavier_uniform_